build/examples/Makefile

```
@@ -0,0 +1,197 @@
+# Prepare targets for installation.
+preinstall/fast:
+	cd /Users/rhseung/Documents/GitHub/hinguri-pingpong/build && $(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 examples/preinstall
+.PHONY : preinstall/fast
+
+# clear depends
+depend:
+	cd /Users/rhseung/Documents/GitHub/hinguri-pingpong/build && $(CMAKE_COMMAND) -S$(CMAKE_SOURCE_DIR) -B$(CMAKE_BINARY_DIR) --check-build-system CMakeFiles/Makefile.cmake 1
+.PHONY : depend
+
+# Help Target
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (11).

(maxbodylength)